plane/source/docs/automatic-landing.rst

```diff
@@ -300,279 +300,234 @@ minimise barometric error problems with auto-land
+-  *Mode change*. For human piloted landing abort you can switch out of AUTO mode into, for example MANUAL/STABILIZE/FBWA, and navigate the aircraft safely however you'd like. Using this method will skip abort behavior step 2 because it is being done manually. When switching back to AUTO the mission will resume as described in step 3 below.
+
+
+Step 2) Abort land flight behavior
 ----------------------------------
-
-The abort behaviour depends on the trigger method, your mission, and
-other factors.
-
-User-defined abort sequence
-+++++++++++++++++++++++++++
-
-If an abort is triggered using the ``CMD_DO_GO_AROUND`` command, the
-mission will increment to the command after the LAND command, and
-continue as normal. As the command sequence after LAND is arbitrary,
-this approach allows a fully customized abort sequence.
-
-.. tip::
-
-   A useful mission item to put directly after LAND is the
-   :ref:`CONTINUE_AND_CHANGE_ALT <mav_cmd_nav_continue_and_change_alt>`
-   mission item with the first param either 0 or 1. This will cause the
-   aircraft to climb to the desired altitude.
-
-If an abort is triggered by changing modes or increasing throttle to
->90%, Plane will check if the command after LAND is
-``CONTINUE_AND_CHANGE_ALT``. If it is, then Plane will execute that
-command, and then continue the mission when it reaches the desired
-altitude.  Otherwise Plane will follow the default abort sequence below.
-
-Fallback/default abort sequence
-+++++++++++++++++++++++++++++++
-
-If an abort is triggered by changing modes (toggle out of AUTO to FBWA
-and then back to AUTO) or increasing throttle to >90% and
-``CONTINUE_AND_CHANGE_ALT`` is **not found** after the LAND command,
-then Plane will attempt to auto-climb to a safe altitude using
-best-available information.
-
-Plane uses :ref:`TKOFF_THR_MAX <TKOFF_THR_MAX>`
-for the throttle value. The target altitude and pitch used are the first
-values out of the following that are found/defined:
-
-#. ``LAND`` command abort altitude (``param 1``).
-#. ``TAKEOFF`` command altitude and pitch value.
-#. Default altitude of 30m and 10 degree pitch (used if the ``LAND``
-   abort altitude not defined and the mission has no ``TAKEOFF``
-   command).
-
-The climb-out procedure is complete when the abort altitude is reached.
-Plane will then change the mission index to allow for a predictable
-landing approach:
-
--  Jump to a ``DO_LAND_START`` command if there is one in the mission.
--  Otherwise decrement the mission index to restart the glide slope.
-
+The abort behaviour has a default configuration and does not require a pre-planned mission or special configuring. The default abort behavior is to simulate an auto-takeoff: pitch up at least 10 degrees and set throttle to TKOFF_THR_MAX and hold the heading until it reaches a target altitude of 30m. It is possible to override the pitch and altitude to allow for a customized behavior.

```

Comment:
"special configuring" sounds wrong. Probably 
    
    The abort behaviour has a default configuration (by default there is no requirement for a pre-planned mission or special configuration).